gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageNativeFileSystem.java

```diff
@@ -91,15 +113,17 @@ public GoogleCloudStorageNativeFileSystem(
    */
   @VisibleForTesting
   public GoogleCloudStorageNativeFileSystem(
```

Comment:
SInce the method is only used for testing, and we are modifying the signature. Can we make it package-private ?